Barotrauma/BarotraumaShared/SharedSource/StatusEffects/StatusEffect.cs

```diff
@@ -2281,15 +2285,74 @@ void OnItemSpawned(Item newItem, ItemSpawnInfo itemSpawnInfo)
             }
             if (removeCharacter)
             {
+                void SpawnDespawnContainer(Character character)
+                {
+                    ItemPrefab containerPrefab;
+                    if (saveItemsContainerIdentifier == null) // If no custom identifier is specified for the container item, use the default ones.
+                    {
+                        containerPrefab =
+                        ItemPrefab.Prefabs.Find(me => me.Tags.Contains("despawncontainer")) ??
+                        (MapEntityPrefab.FindByIdentifier("metalcrate".ToIdentifier()) as ItemPrefab);
+                    }
+                    else
+                    {
+                        containerPrefab = ItemPrefab.Prefabs.Find(me => me.Identifier == saveItemsContainerIdentifier);
+                    }
+
+                    if (containerPrefab == null)
+                    {
+                        if (saveItemsContainerIdentifier == null)
+                        {
+                            DebugConsole.NewMessage("Could not spawn a container for a removed character's items. No item with the tag \"despawncontainer\" or the identifier \"metalcrate\" found.", Color.Red);
```

Comment:
In my opinion this should use DebugConsole.ThrowError, it's a severe enough issue that it should pop up the debug console